tasks/carry_my_luggage/src/carry_my_luggage/state_machine.py

```diff
@@ -5,10 +5,23 @@
 
 import rospy
 from lasr_person_following.msg import FollowAction
+from sensor_msgs.msg import Image
 
 
 class CarryMyLuggage(smach.StateMachine):
 
+    class GetImage(smach.State):
```

Comment:
Is there a reason this is here rather than using the existing get_image skill?

Reply:
Just forgot that it existed! Good spot